kolibri/plugins/learn/assets/src/vue/toolbar/index.vue

```diff
@@ -0,0 +1,238 @@
+<template>
+
+  <div v-bind:class="['toolbar-show', displayToolBar ? 'toolbar-hide' : '' ]" >
```

Comment:
I know you are still working on it, just wanted to mention that you should make sure the visual order of elements in the toolbar corresponds to the tabbing order (which is dependent on their position in the code). In current master, for example, channel switcher drop-down `select` is below the search button component in the code, yet the visual order is the other way around, which can cause confusion if user navigates by keyboard. 

So if the visual order in the toolbar is `breadcrumbs - channel select - search button`, make sure that focus by TAB key follows the same order.